src/components/datepicker/js/calendarMonth.js

```diff
@@ -156,6 +159,41 @@
     );
   };
 
+  /**
+   * Change the selected date in the calendar (ngModel value has already been changed).
+   * @param {Date} date
+   */
+  CalendarMonthCtrl.prototype.changeSelectedDate = function(date) {
```

Comment:
Where did this come from?

Reply:
I probably resurrected this by accident in a rebase